### ISSUE TITLE
Document `chimera-prunekernels`

### ISSUE THREAD

**Issue by triallax** [chimera-linux/chimera-linux.github.io#38]
we get asked about this quite often on irc

**Comment by sidkshatriya** [comment]
For now we can simply say that it is a shell script and curious people can inspect it:

`$EDITOR /bin/chimera-prunekernels`

**Comment by nekopsykose** [comment]
writing documentation that says 'this program is a shell script, look at it yourself' is not particularly friendly or useful to whoever is reading the documentation, it's the same as just not having any docs at all

**Comment by triallax** [comment]
i agree, i was thinking about actually explaining what the script does and showing some example usages
